openedx/core/djangoapps/user_api/tests/test_views.py

```diff
@@ -1020,6 +1021,10 @@ def test_register_form_third_party_auth_running(self):
                 }
             )
 
+    def test_register_form_platform_name(self):
```

Comment:
I'm not quite sure how this test relates to your change - is it testing that platform name comes through to the register form?